extensions/arc/src/ui/dashboards/postgres/postgresParametersPage.ts

```diff
@@ -536,6 +536,7 @@ export class PostgresParametersPage extends DashboardPage {
 			this.parameterContainer!.addItem(this._parametersTableLoading!);
 		} else {
 			this.parameterContainer!.addItem(this.parametersTable!);
+			this.refreshParametersTable();
```

Comment:
This would also mean the searchbox and restetAllButton is not enabled since it requires the connectToServerButton. Should add `this.searchBox!.enabled = true` and `this.resetAllButton!.enabled = true` here as well.

Reply:
Good to know - I'll let you take care of fixing that up and testing it out 😄